### ISSUE TITLE
[Feature request] Don't add timestamps to restored chat history

### ISSUE THREAD

**Issue by linaori** [funkydude/BasicChatMods#64]
## Describe the problem
The timestamp is added for each message in the history, so you'll see `<timestamp history loaded><timestamp from history>`, and this is done for each load. I've had messages with 7 timestamps 😁 

![image](https://user-images.githubusercontent.com/1754678/198876578-046bbd77-9b9b-45be-ad0f-aeed570a0d20.png)

### What steps will reproduce the problem?

1.  Enable Both the timestamps and history
2.  Reload a couple of times

### What version of the addon are you using?
v10.0.0

**Comment by funkydude** [comment]
This is essentially "working as intended" as the timestamps and history modules have no awareness of each other.

Attempting to resolve this would require string parsing and guessing what should or shouldn't be trimmed from a string, which is impossible to be 100% accurate i.e. there's a chance you cut out something you shouldn't, added by some other addon.

We also want to preserve the original timestamp for people showing seconds, so you'd be in a situation of trying to preserve the original timestamp, adding a new one, and removing everything in between,

In other words, it's just not worth resolving for something most people will never encounter. Having the raw unedited text is more important.

**Comment by linaori** [comment]
For me it would be enough to simply not add the new timestamp to those lines, no parsing needed, just has to not add new timestamps while loading the history

**Comment by linaori** [comment]
It seems I was mistaken, there is a setting in Leatrix Plus that restores the chat messages after a reload, and seems to be conflicting with the restore from this addon. When I disable the Leatrix Plus setting I see no duplicate timestamps.
